### DISCUSSION TITLE
power down the create 3, getting started

### DISCUSSION THREAD

**Discussion by jbhFriends** [iRobotEducation/create3_docs#471] [Questions] [answered]
### How are you connecting to your Create 3?

I'm not sure

### Computer(s) Model(s) and Operating System(s)

dell - windows

### Which version of ROS 2 is installed on your computer?

None

### Which firmware version is installed on your robot?

2.3

### Which RMW is your robot running?

I don't know

### Are there multiple Create 3 robots connected to your network?

None

### Is multicast enabled?

None

### What is the Adapter Board's USB/BLE Toggle currently switched to?

None

### Describe your question.

does the glowing white ring, status of the robot is that the "off" status?
how to power down properly? holding the center button 7 seconds doesn't work

learning to use (day1 today) create 3 for robotics

**Comment by jbhFriends** [comment]
also looking to connect with people who use these robots to learn about them.

**Reply by brianabouchard** [reply]
Welcome @jbhFriends!

**Comment by shamlian** [comment]
Please remove the robot from the dock and hold the center/power button for ten seconds. The robot should power down. You might find more useful information in the [documentation](https://iroboteducation.github.io/create3_docs/hw/face/).

**Reply by jbhFriends** [reply]
on the charging dock the green light comes on for a minute or so and then goes out. is this an issue? are we charging correctly?

what is the small red and black cable with a white connector, which comes in the box? the ends of the red and black wires are just open wire.

**Reply by jbhFriends** [reply]
one of the robots has a red circle, is that mean it's near empty, the charging dock green light is not on, is that an issue?

**Reply by jbhFriends** [reply]
i have held the center button many times, at least 10 sec. nothing changes, glowing white circle

**Reply by brianabouchard** [reply]
Hi @jbhFriends! 

- The red and black cable in the box plugs into the unregulated battery port on the adapter board so you can power a device from the Create 3 robot's battery. For details on what the port can do, check out [this page](https://iroboteducation.github.io/create3_docs/hw/adapter/#unregulated-battery-port). 
- The dock light coming on for a minute or so and then turning off is normal. If your robot is properly charging while on the dock,  you will be able to tell by the light on the robot circling white. See [this page](https://iroboteducation.github.io/create3_docs/hw/face/) for details on what the robot's lights indicate. 
- What happens when you remove the robot from the dock and then hold the button down? Perhaps you can share a video with us so we can see what's going on?

**Reply by jbhFriends** [reply]
yes we did read those, before writing you

**Comment by jbhFriends** [comment]
after about 15 seconds, the light goes off, but then comes back on after a few seconds

**Comment by jbhFriends** [comment]
https://github.com/iRobotEducation/create3_docs/assets/144817110/b0aaaf3d-fd0e-42be-9c6e-d4d60dd4deb0

**Reply by jbhFriends** [reply]
both our two create3s have same behavior

**Reply by jbhFriends** [reply]
just now, by itself, it went off. then came back on

**Reply by brianabouchard** [reply]
Ok, at any point when the robot is on does the white light stop spinning? So we can try to troubleshoot a bit better, do you recall what you were asking the robots to do when they went into this state?

**Reply by jbhFriends** [reply]
[2024 Botball Create 3 Slides.pdf](https://github.com/iRobotEducation/create3_docs/files/13591300/2024.Botball.Create.3.Slides.pdf)

**Reply by jbhFriends** [reply]
i was following these instructions to connect a controller to it.

**Reply by jbhFriends** [reply]
so, i had pushed the 1 and 2 combo to get it to be seen by my wifi. then i did update the firmware to 2.3 as instructed, then i connected the controller, so now if i connect to my controller, i can then pull up the create3 via an IP in the browser.

also, there were two small changes to the settings, you will see 
safety ovrride from none to full
reflexes_enabled:false

then i save it. pretty sure it went through, was able to access through the controller, when both were on. 
so at this point, just want to power the thing down. both don't respond as intended, which i'm showing in the video

**Reply by brianabouchard** [reply]
Ok, that's great that you are able to communicate with the Create 3 via the webserver. Can you do me a favor and try going to the webserver via the IP in a browser and then under the application menu at the top, click reboot robot? Please do all of this while the robot is not on the dock. Then wait for the white light to stop spinning, this can take some time. Then try holding down the power button.

**Reply by jbhFriends** [reply]
ok, i will try this. faculty meeting now, will get to try it and a response to you in the morning . thank you so much

**Reply by brianabouchard** [reply]
Great, let me know. Also, if that does not work, please upload the robot's logs so we can take a look at any errors it might be throwing.

**Reply by jbhFriends** [reply]
IP connection works as yesterday, through our controller wifi,  (it's off the dock)
but the CREATE3  under "application" --> "restart application" we assume you meant that?
nowhere is there a "reboot robot", just wanted to be specific
so after restart application, the light at no point was off, 
and the button has no effect
the light glowing white ring

**Reply by jbhFriends** [reply]
[messages(1).txt](https://github.com/iRobotEducation/create3_docs/files/13603072/messages.1.txt)

the log after the "restart" application

**Reply by jbhFriends** [reply]
![Screenshot 2023-12-07 at 12-13-45 Provision](https://github.com/iRobotEducation/create3_docs/assets/144817110/f7e15dab-cd5b-4344-bc60-dc10911120e3)

**Reply by jbhFriends** [reply]
just showing our menu bar

**Reply by brianabouchard** [reply]
Ok, thanks for this additional information. From the logs, it looks like there is something wrong with ROS 2 parameter yaml file that was edited on the application configuration page, which is causing the application to crash loop. Can you please post the contents of that text box on the web server or a screenshot? It's the bottom half of the application configuration page.

**Reply by jbhFriends** [reply]
that would explain why my two bots are exactly same in their dysfunction.. i will look

**Reply by brianabouchard** [reply, marked as answer]
Ah ok! You need a space in the last line between "reflexes_enabled:" and "false". Once you make that change, restart the application again and hopefully it will stop the endless spinning light :)

**Reply by jbhFriends** [reply]
it's also not it quotes, I should put quotes and space? will do

**Reply by jbhFriends** [reply]
so now it has shifted to a solid red ring

**Reply by jbhFriends** [reply]
after saving the new piece

**Reply by jbhFriends** [reply]
i see the change, when i have reload the page

**Comment by jbhFriends** [comment]
thanks for any help. we just learning how to use

**Reply by jbhFriends** [reply]
ok, so now I have exactly replicated this behavior on the second create.
immediately changes to solid red
--- so i will wait for you reply.  do i need no quotes around the false?  Or is this now the expected behavior? the website says solid red is error state, right? so maybe just i'll try the other way and see

**Reply by jbhFriends** [reply]
oh now I got a chime and a solid white, not glowing. So that was nice

**Reply by jbhFriends** [reply]
ok, so yay, i have two sleeping robots. what can i do with them :)

**Reply by slowrunner** [reply]
That Light Ring can tell lots:

Create 3 Light Ring While Charging:
- Spinning White: booting - wait for happy sound
- Partial White: Charging
- Solid White: Fully Charged
- Pulsing Red: Battery < 10%

Create3 Light Ring - While Not Charging:
- Spinning White: booting - wait for happy sound
- Solid White: Power is On - Operational (default)
- Pulsing Red: Battery < 10%, place on charger
- Solid Red: Robot Error, cycle power 
- Half Solid Orange: Back-up safety activated
- Half Solid Yellow: Wheels Disabled

**Reply by shamlian** [reply]
[Here's the light ring not-so-secret decoder ring.](https://iroboteducation.github.io/create3_docs/hw/face/#light-ring)

**Comment by jbhFriends** [comment]
did I say thank you?
Thank you

**Comment by jbhFriends** [comment]
question; when we take the thing off the charging dock, is it on?
we show white solid ring.

**Reply by brianabouchard** [reply]
Yes, if you have a solid white ring, it is on.
